onnx/backend/test/cpp/driver_test.cc

```diff
@@ -199,8 +210,8 @@ class ONNXCppDriverTest
         GTEST_SKIP();
         return;
       }
-      ASSERT_EQ(is_compatible, ONNXIFI_STATUS_SUCCESS);
-      ASSERT_EQ(
+      ASSERT_MEMORYSAFE_SUCCESS(is_compatible, false);
```

Comment:
why do we specifically care about releasing graph but not onnxReleaseBackend or onnxReleaseBackendID?

Reply:
In fact we do care about releasing backend, backendID and free memory. However, they are done in TEST_P() instead of in this test class, therefore even the ASSERT failed in class, the release of backend would still be excuted. I have tested this mechanism myself.

Reply:
(And also even we dont release&free, OS would always do these things for us. quote: a google testing engineer.... Considering to uninstall all google product in my computer...

Reply:
"And also even we dont release&free, OS would always do these things for us" This is true, but ASAN would often report leaks in this cases.